File: evidence/generated-systems/angular/healthcare-on-angular/src/app/models/LabResult.js

// Define collection and schema for LabResult
export interface LabResult {
    resultCode:
	type : string
    issuedDate:
	type : Date
    LaboratoryOrder:
	type : Schema.Types.ObjectId
    Observations:
 	type : [{ type: Schema.Types.ObjectId, ref: 'Observation' }]
    Laboratory:
	type : Schema.Types.ObjectId
    Status:
 	type : String
#
    collection: 'labResults'
}

File: evidence/generated-systems/rubyonrails/healthcare-on-rails/healthcareonrails/app/models/labresult.rb
class LabResult < ApplicationRecord
  enum Status: [:Registered, :Partial, :Final, :Corrected, :Cancelled]


  has_many :LaboratoryOrder, class_name: 'LaboratoryOrder'
  has_many :Observations, class_name: 'Observation'
  has_many :Laboratory, class_name: 'Laboratory'

end
